tsdb/functions.go

```diff
@@ -274,11 +276,11 @@ func MapCount(input *MapInput) interface{} {
 	return n
 }
 
-type InterfaceValues []interface{}
+type interfaceValues []interface{}
```

Comment:
The type is only used in the tsdb package and doesn't really look like something that should necessarily be exported, so I went by the suggestion to "favor making symbols private where possible before documenting.". 

Looking at the code more closely, I see that it's used inside the ReduceDistinct function and one could argue that it's needed to call ReduceDistinct with values that were not returned by another function from this package.